metaflow/plugins/kfp/kfp.py

```diff
@@ -96,48 +103,32 @@ def initial_setup_op_func(code_url: str)  -> StepOutput:
 
     print("\n----------RUNNING: METAFLOW INSTALLATION----------")
     subprocess.call(["pip3 install --user --upgrade {modified_metaflow_git_url}".format(
-        modified_metaflow_git_url=MODIFIED_METAFLOW_URL)],
-                    shell=True)
+        modified_metaflow_git_url=MODIFIED_METAFLOW_URL)], shell=True)
 
-    print("\n----------RUNNING: MAIN STEP COMMAND--------------")
+    print("\n----------RUNNING: INIT COMMAND-------------------")
     S3_BUCKET = os.getenv("S3_BUCKET")
     S3_AWS_ARN = os.getenv("S3_AWS_ARN")
     S3_AWS_REGION = os.getenv("S3_AWS_REGION")
 
     define_s3_env_vars = 'export METAFLOW_DATASTORE_SYSROOT_S3="{}" && export METAFLOW_AWS_ARN="{}" ' \
                          '&& export METAFLOW_AWS_S3_REGION="{}"'.format(S3_BUCKET, S3_AWS_ARN, S3_AWS_REGION)
     define_username = 'export USERNAME="kfp-user"'
```

Comment:
let's add a todo, how does user Metaflow user name map to KF world?

Reply:
Done